Anytype/Sources/PresentationLayer/Search screen/InternalViewModels/SpaceSearchViewModel/SpaceSearchViewModel.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import SwiftUI
+import Combine
+
+final class SpaceSearchViewModel: SearchViewModelProtocol {
```

Comment:
@MainActor?

Reply:
Need to refactor a bit all SearchView & SearchViewModel to resolve nonisolated warnings. Need to be done in whole scope.